packages/lib/server/queries/booking/index.ts

```diff
@@ -10,11 +10,12 @@ export const getTotalBookingDuration = async ({
   endDate: Date;
 }) => {
   // Aggregates the total booking time for a given event in a given time period
+  // FIXME: bookings that overlap on one side will never be counted
   const [totalBookingTime] = (await prisma.$queryRaw`
     SELECT SUM(EXTRACT(EPOCH FROM ("endTime" - "startTime")) / 60) as "totalMinutes"
     FROM "Booking"
     WHERE "status" = 'accepted'
-      AND "id" = ${eventId}
+      AND "eventTypeId" = ${eventId}
       AND "startTime" >= ${startDate}
       AND "endTime" <= ${endDate};
```

Comment:
Using the wrong key caused this to return `0` most of the time, and a random booking duration at best. Need some input here re counting overlapping bookings, e.g. starts in one week and ends in the next. Currently, those bookings are never considered when checking weekly limits. Same naturally applies to other periods.